### DISCUSSION TITLE
How to complete a pull transfer process

### DISCUSSION THREAD

**Discussion by jannotti-glaucio** [eclipse-edc/Connector#4594] [Q&A] [answered]
Guys, in a pull transfer the transfer process keeps in started status infinitely, either after the consumer calls the /public on the provider to get the transfer data. How can I change the pull transfer process to completed ? I only found on the management API an endpoint to terminate the transfer, but not one to successfully complete it.

Regards,

**Comment by ndr-brt** [comment, marked as answer]
Hi, I suggest to look at the documentation: https://eclipse-edc.github.io/documentation/for-contributors/control-plane/policy-monitor/
